app/server/lib/FlexServer.ts

```diff
@@ -405,19 +405,23 @@ export class FlexServer implements GristServer {
 
   public addLogging() {
     if (this._check('logging')) { return; }
-    if (process.env.GRIST_LOG_SKIP_HTTP) { return; }
+    if (isAffirmative(process.env.GRIST_LOG_SKIP_HTTP)) { return; }
     // Add a timestamp token that matches exactly the formatting of non-morgan logs.
     morganLogger.token('logTime', (req: Request) => log.timestamp());
     // Add an optional gristInfo token that can replace the url, if the url is sensitive.
     morganLogger.token('gristInfo', (req: RequestWithGristInfo) =>
                        req.gristInfo || req.originalUrl || req.url);
     morganLogger.token('host', (req: express.Request) => req.get('host'));
-    const msg = ':logTime :host :method :gristInfo :status :response-time ms - :res[content-length]';
+    morganLogger.token('body', (req: express.Request) => JSON.stringify(req.body));
+    const msg = ':logTime :host :method :body :gristInfo :status :response-time ms - :res[content-length]';
     // In hosted Grist, render json so logs retain more organization.
     function outputJson(tokens: any, req: any, res: any) {
       return JSON.stringify({
         timestamp: tokens.logTime(req, res),
         method: tokens.method(req, res),
+        body: isAffirmative(process.env.GRIST_LOG_HTTP_BODY)
```

Comment:
Maybe this could be writtent differently to avoid outputting `body: undefined` when `GRIST_LOG_HTTP_BODY` is not enabled, e.g. 
```suggestion
        ...(isAffirmative(process.env.GRIST_LOG_HTTP_BODY) ? { body : tokens.body(req, res) } : {})
```